From 5ba555788defd974b9c8afc1f974547449eafb80 Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Mon, 11 Jun 2018 13:54:29 +0300
Subject: [PATCH 3968/5242] MLK-18373 imx8qm: A72 1300mhz is actually 1296mhz

commit  4c0ebc0484bdb4d34f60760fd754844f613a0ee8 from
https://source.codeaurora.org/external/imx/linux-imx.git

Setting a72 clock to 1300mhz and reading back the value from clk reveals
the rate is actually 1296mhz:

root@imx8mmevk:~# cpufreq-set -c 4 -f 1300000
root@imx8mmevk:~# cat /sys/kernel/debug/clk/clk_summary |grep a72
 a72_div                                  0            0  1296000000          0 0

This causes some cpufreq tests to fail. Fix by setting OPP to 1296000.

Fixes: 2b6d66acdd71 ("MLK-18331 ARM64: dts: freescale: imx8qm: update cpufreq set points")

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Acked-by: Anson Huang <anson.huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 72d7290..604ad51 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -199,7 +199,7 @@
 		/* voltage is maintained by SCFW, so no need here */
 		1596000	   0
 		1464000	   0
-		1300000	   0
+		1296000	   0
 		1056000	   0
 		600000     0
 	>;
-- 
1.7.9.5

